### DISCUSSION TITLE
Problems with usage of Abaqus mesh in Global strain calculations

### DISCUSSION THREAD

**Discussion by abarun22** [idaholab/moose#19945] [Q&A General] [answered]
Dear all,
I am running a multiapp problem with my sub-app considered a submodel of the main model. The 'submodel' effectively is a cut out of the main model sharing the same spatial coordinates. It essentially acts as an RVE enforcing the strain periodicity conditions in the model. Eventually I am interested in coupling these two models together at each load step and thinking to update the deformation of the main model based on the micro-beheviour of the submodel. Just for the case of proper creation of the submodel I had to use ABAQUS for modelling the FE. However the submodel runs an empty simulation with ABAQUS input file and i am getting the following lines.
<code><pre>
sub0: Time Step 1, time = 0.01, dt = 0.01
sub0:  0 Nonlinear |R| = 0.000000e+00
sub0:  Solve Converged!
</code></pre>
  I need to make couple of adjustments in the input file when i am using an ABAQUS mesh. The 'auto_direction' command in periodic BC is unusable and so i manually defined and matched the periodic boundaries. A nodeset consisting of a single node (node closest to centroid) created to model the reference point needed for constraining the model. The attached files could used as a reference to diagnose the problem. Any advise on fixing the problem would be highly appreciated.
[04012022.zip](https://github.com/idaholab/moose/files/7807710/04012022.zip)

**Comment by GiudGiud** [comment, marked as answer]
@dschwen on Abaqus

**Reply by GiudGiud** [reply]
@abarun22 is this solved?

This 0 residual means there is nothing to solve, we're already at the solution. What is imposing a load / disturbing the equilibrium?
